application/src/main/java/org/togetherjava/tjbot/commands/moderation/BanCommand.java

```diff
@@ -192,5 +200,22 @@ public void onSlashCommand(@NotNull SlashCommandEvent event) {
                 .requireNonNull(alreadyBanned.getFailure()), event, guild, target).orElseGet(
                         () -> banUserFlow(target, author, reason, deleteHistoryDays, guild, event));
         }).queue();
+
+        try {
+            database.write(context -> {
+                BanSystemRecord banSystemRecord = context.newRecord(BanSystem.BAN_SYSTEM)
+                    .setUserid(target.getIdLong())
+                    .setAuthorId(author.getIdLong())
+                    .setGuildId(guild.getIdLong())
+                    .setIsBanned(true)
+                    .setBanReason(reason);
+                if (banSystemRecord.update() == 0) {
+                    banSystemRecord.insert();
+                }
+            });
+            logger.info("Saved the user '{}' to the ban system.", target.getAsTag());
```

Comment:
obsolete log message. make it `debug` and not `info`

Reply:
the log also reads kinda confusing to me. who is the "ban system". you probably wanted to say something like u stored the ban in the database.